src/feature/featurelist.h

```diff
@@ -202,6 +202,13 @@ FEATURE(startOnBoot,            // Feature ID
         QStringList(),          // feature dependencies
         FeatureCallback_startOnBoot)
 
+FEATURE(stagingUpdateServer,        // Feature ID
+        "Staging Update Server",    // Feature name
+        FeatureCallback_hasBalrog,  // Can be flipped on
+        FeatureCallback_false,      // Can be flipped off
```

Comment:
Does this mean that once the feature is enabled it can't be disabled?

Reply:
I don't really understand the logic behind our feature flagging methods, but it can be turned off just fine. However, when I replace this with `FeatureCallback_true` it resulted in a default-enabled feature which we definitely don't want here.